src/actions/job.ts

```diff
@@ -23,8 +24,31 @@ export async function getJobInfo(octokit: Octokit): Promise<JobInfo> {
                     per_page: PAGE_SIZE,
                     page
                 })
-            } catch (error) {
+            } catch (error: any) {
                 result = undefined
+                logger.info(
```

Comment:
`debug` level is better

Reply:
I think, it is better to log inside `if` statement at `debug` level

Reply:
done